akvo/rsr/models/project_editor_validation.py

```diff
@@ -49,14 +49,14 @@ class ProjectEditorValidation(Model):
     """
     MANDATORY_ACTION = 1
     HIDDEN_ACTION = 2
-    READ_ONLY_ACTION = 3
+    # READ_ONLY_ACTION = 3
```

Comment:
I'd vote for an issue or a roadmap card with a link to this commit as a better reminder.  Sorry for the nit-pick!